org.eclipse.scanning.example/src/org/eclipse/scanning/example/malcolm/ExampleMalcolmDevice.java

```diff
@@ -146,26 +146,30 @@ public void request(PVStructure args, RPCResponseCallback callback)
                     return;
                 }
                 
-                if (methodName.equals("configure")) {
+
+                Structure mapStructure = fieldCreate.createFieldBuilder().
+            			setId("malcolm:core/Map:1.0").
+            			createStructure();
+                PVStructure returnPvStructure = pvDataCreate.createPVStructure(mapStructure);
+                
+                if (methodName.equals("validate")) {
+                	returnPvStructure = args;
+                } else if (methodName.equals("configure")) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move the "configure" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1132)